alcs-frontend/src/app/services/application/application-parcel/application-parcel.service.ts

```diff
@@ -21,4 +21,20 @@ export class ApplicationParcelService {
       throw e;
     }
   }
+
+  async setParcelArea(uuid: string, alrArea: number | null): Promise<ApplicationParcelDto[]> {
+    try {
+      const res = await firstValueFrom(
+        this.http.post<ApplicationParcelDto[]>(`${this.baseUrl}/${uuid}`, {
+          alrArea,
+        })
+      );
+      this.toastService.showSuccessToast('Application updated');
```

Comment:
nitpick, "Application parcel updated" so will match the error toast?

Reply:
yeah this was a bit odd, but the requirements specifically say to say Application updated when the ALR area is saved.